code/datums/mind/_mind.dm

```diff
@@ -168,7 +168,14 @@
 
 /datum/mind/proc/get_language_holder()
 	if(!language_holder)
```

Comment:
if you want to go the distance you could do this

```suggestion
	if(isnull(language_holder))
```

just writing this since it's not 24 hours yet and if someone else just wants to yeet this through it's okay, just figured i'd mention it